src/definitions/pinghf.json

```diff
@@ -0,0 +1,465 @@
+                        "units": "mm/s"
+                    }
+                ]
+            },
+            "range": {
+                "id": "2204",
+                "description": "The scan range for acoustic measurements. Measurements returned by the device will lie in the range (scan_start, scan_start + scan_length).",
+                "payload": [
+                    {
+                        "name": "scan_start",
+                        "type": "u32",
+                        "description": "The beginning of the scan range in mm from the transducer.",
+                        "units": "mm"
+                    },
+                    {
+                        "name": "scan_length",
+                        "type": "u32",
+                        "description": "The length of the scan range.",
+                        "units": "mm"
+                    }
+                ]
+            },
+            "ping_interval": {
+                "id": "2206",
+                "description": "The interval between acoustic measurements.",
+                "payload": [
+                    {
+                        "name": "ping_interval",
+                        "type": "u16",
+                        "description": "The minimum interval between acoustic measurements. The actual interval may be longer.",
+                        "units": "ms"
+                    }
+                ]
+            },
+            "gain_setting": {
+                "id": "2207",
+                "description": "The current gain setting.",
+                "payload": [
+                    {
+                        "name": "gain_setting",
+                        "type": "u32",
+                        "description": "The current gain setting. The analog gain can be determined by requesting the analog_gain_levels message."
+                    }
+                ]
+            },
+            "transmit_duration": {
+                "id": "2208",
+                "description": "The duration of the acoustic activation/transmission.",
+                "payload": [
+                    {
+                        "name": "transmit_duration",
+                        "type": "u16",
+                        "description": "Acoustic pulse duration.",
+                        "units": "microseconds"
+                    }
+                ]
+            },
+            "general_info": {
+                "id": "2210",
+                "description": "General information.",
+                "payload": [
+                    {
+                        "name": "firmware_version_major",
+                        "type": "u16",
+                        "description": "Firmware major version."
+                    },
+                    {
+                        "name": "firmware_version_minor",
+                        "type": "u16",
+                        "description": "Firmware minor version."
```

Comment:
We could remove this functions, since we the the common one.

Reply:
i'll take it out.